specification/storage/data-plane/Microsoft.FileStorage/preview/2020-10-02/file.json

```diff
@@ -5045,6 +5072,30 @@
                     "description": "Content length of the file. This value may not be up-to-date since an SMB client may have modified the file locally. The value of Content-Length may not reflect that fact until the handle is closed or the op-lock is broken. To retrieve current property values, call Get File Properties.",
                     "type": "integer",
                     "format": "int64"
+                },
+                "CreationTime": {
+                    "type": "string",
+                    "format": "date-time-rfc1123"
+                },
+                "LastAccessTime": {
+                    "type": "string",
+                    "format": "date-time-rfc1123"
+                },
+                "LastWriteTime": {
+                    "type": "string",
+                    "format": "date-time-rfc1123"
+                },
+                "ChangeTime": {
+                    "type": "string",
+                    "format": "date-time-rfc1123"
+                },
```

Comment:
I mean these all should be iso-8601 except last-modified should be rfc-1123

Reply:
@gapra-msft, is this an issue in Java?

Reply:
I think @jaschrep-msft had to add a manual transform, he can confirm but I think yes

Reply:
Here is how iso-8601 is generated in blob swagger
https://github.com/Azure/azure-rest-api-specs/blob/storage-dataplane-preview/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json#L10108